x-pack/plugins/watcher/public/application/sections/watch_status_page/components/execution_history_panel.tsx

```diff
@@ -314,7 +314,7 @@ export const ExecutionHistoryPanel = () => {
                 <FormattedMessage
                   id="xpack.watcher.sections.watchHistory.watchHistoryDetail.title"
                   defaultMessage="Executed on {date}"
-                  values={{ date: watchHistoryDetails.startTime }}
+                  values={{ date: watchHistoryDetails.startTime?.format() }}
```

Comment:
Optional operator is included because `watchHistoryDetails.startTime` doesn't always exist, especially on the initial render